src/pybind/mgr/cephadm/serve.py

```diff
@@ -965,6 +965,16 @@ def _ok_to_stop(remove_daemons: List[orchestrator.DaemonDescription]) -> bool:
                 break
         return daemons_to_remove
 
+    def remove_given_daemons(self, daemons_to_remove: List[orchestrator.DaemonDescription]) -> Tuple[bool, Set[str]]:
+        r = False
```

Comment:
This `r` value looks odd to me. Why can't hosts_altered being empty or not determine this "truthiness?"